components/docs-chef-io/content/automate/restore.md

```diff
@@ -62,6 +62,15 @@ Ensure access for the backup type used:
          # Use "m" for megabytes and "g" for gigabytes
      ```
 
+{{< note >}}
+
+- If you are restoring the backup taken from an older version of Automate, then you need to provide the `--airgap-bundle </path/to/current/bundle>` with your restore command.
```

Comment:
Replace it with: - If restoring the backup from an older version of Automate; you must provide the `--airgap-bundle </path/to/current/bundle>` with your restore command.

Reply:
Done